forum/views.py

```diff
@@ -117,53 +117,54 @@ def deepValidateAndFetch(request, errors):
         params['title'] = form.cleaned_data['title']
     except:
         params['title'] = ""
-        errors.append({ "field": "title", "msg" :"Le titre du sujet ne peut pas être vide"})
+        errors.append({"field": "title", "msg": "Le titre du sujet ne peut pas être vide"})
 
     try:
         params['visibdata'] = form.cleaned_data['visibdata']
     except:
         params['visibdata'] = ""
-        errors.append({ "field": "visibdata", "msg" :"Le paramètre de visibilité ne peut pas être vide"})
+        errors.append({"field": "visibdata", "msg": "Le paramètre de visibilité ne peut pas être vide"})
```

Comment:
"visibdata", not the clearest choice of naming

Reply:
Not my code